src/documentation/runners/spark.md

```diff
@@ -6,4 +6,115 @@ redirect_from: /learn/runners/spark/
 ---
 # Using the Apache Spark Runner
 
-This page is under construction ([BEAM-507](https://issues.apache.org/jira/browse/BEAM-507)).
+The Apache Spark Runner can be used to execute Beam pipelines using [Apache Spark](http://spark.apache.org/). 
+The Spark Runner can execute Spark pipelines just like a native Spark application; deploying a self-contained application for local mode, running on Spark's Standalone RM, or using YARN or Mesos.
+
+The Spark Runner executes Beam pipelines on top of Apache Spark, providing:
+
+* Batch and streaming (and combined) pipelines.
+* The same fault-tolerance [guarantees](http://spark.apache.org/docs/1.6.3/streaming-programming-guide.html#fault-tolerance-semantics) as provided by RDDs and DStreams.
+* The same [security](http://spark.apache.org/docs/1.6.3/security.html) features Spark provides.
+* Built-in metrics reporting using Spark's metrics system, which reports Beam Aggregators as well.
+* Native support for Beam side-inputs via spark's Broadcast variables.
+
+The [Beam Capability Matrix]({{ site.baseurl }}/documentation/runners/capability-matrix/) documents the currently supported capabilities of the Spark Runner.
+
+_**Note:**_ _support for the Beam Model in streaming is currently experimental, follow-up in the [mailing list]({{ site.baseurl }}/get-started/support/) for status updates._
+
+## Spark Runner prerequisites and setup
+
+The Spark runner currently supports Spark's 1.6 branch, and more specifically any version greater than 1.6.0.
+
+You can add a dependency on the latest version of the Spark runner by adding to your pom.xml the following:
+```java
+<dependency>
+  <groupId>org.apache.beam</groupId>
+  <artifactId>beam-runners-spark</artifactId>
+  <version>{{ site.release_latest }}</version>
+</dependency>
+```
```

Comment:
As the Spark runner doesn't provide a BoM (I created another Jira about that), I think end-users have to define the following additional dependencies:

```
                <dependency>
                    <groupId>org.apache.hadoop</groupId>
                    <artifactId>hadoop-common</artifactId>
                    <version>${hadoop.version}</version>
                    <exclusions>
                        <!-- exclude old Jetty version of servlet API -->
                        <exclusion>
                            <groupId>org.mortbay.jetty</groupId>
                            <artifactId>servlet-api</artifactId>
                        </exclusion>
                    </exclusions>
                </dependency>
                <dependency>
                    <groupId>org.apache.hadoop</groupId>
                    <artifactId>hadoop-client</artifactId>
                    <version>${hadoop.version}</version>
                </dependency>
                <dependency>
                    <groupId>org.apache.hadoop</groupId>
                    <artifactId>hadoop-mapreduce-client-core</artifactId>
                    <version>${hadoop.version}</version>
                </dependency>

                <dependency>
                    <groupId>com.fasterxml.jackson.core</groupId>
                    <artifactId>jackson-core</artifactId>
                    <version>${jackson.version}</version>
                </dependency>
                <dependency>
                    <groupId>com.fasterxml.jackson.core</groupId>
                    <artifactId>jackson-annotations</artifactId>
                    <version>${jackson.version}</version>
                </dependency>
                <dependency>
                    <groupId>com.fasterxml.jackson.core</groupId>
                    <artifactId>jackson-databind</artifactId>
                    <version>${jackson.version}</version>
                </dependency>
                <dependency>
                    <groupId>com.fasterxml.jackson.module</groupId>
                    <artifactId>jackson-module-scala_2.10</artifactId>
                    <version>${jackson.version}</version>
                </dependency>
```